gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/AuditServiceImpl.java

```diff
@@ -89,8 +89,10 @@
         Builder criteria = new Builder().from(query.getFrom()).to(query.getTo());
 
         if (query.isCurrentEnvironmentLogsOnly()) {
+            // FIXME: Rework AuditQuery to remove call to GraviteeContext.getCurrentEnvironment() here
             criteria.references(Audit.AuditReferenceType.ENVIRONMENT, Collections.singletonList(GraviteeContext.getCurrentEnvironment()));
         } else if (query.isCurrentOrganizationLogsOnly()) {
+            // FIXME: Rework AuditQuery to remove call to GraviteeContext.getCurrentOrganization() here
```

Comment:
FIXME: Rework AuditQuery to remove call to GraviteeContext.getCurrentOrganization() here